tests/cnf/ran/talm/internal/tsparams/talmvars.go

```diff
@@ -14,12 +20,21 @@ var (
 
 	// ReporterNamespacesToDump tells to the reporter from where to collect logs.
 	ReporterNamespacesToDump = map[string]string{
-		TestNamespace: "",
+		TestNamespace:      "",
+		TemporaryNamespace: "",
 	}
 
 	// ReporterCRDsToDump tells to the reporter what CRs to dump.
 	ReporterCRDsToDump = []k8sreporter.CRData{
-		{Cr: &v1.PodList{}},
+		{Cr: &corev1.PodList{}},
+		{Cr: &corev1.NamespaceList{}},
+		{Cr: &cguv1alpha1.ClusterGroupUpgradeList{}, Namespace: ptr.To(TestNamespace)},
+		{Cr: &cguv1alpha1.PreCachingConfigList{}, Namespace: ptr.To(TestNamespace)},
+		{Cr: &policiesv1.PolicyList{}, Namespace: ptr.To(TestNamespace)},
+		{Cr: &policiesv1.PlacementBindingList{}, Namespace: ptr.To(TestNamespace)},
+		{Cr: &placementrulev1.PlacementRuleList{}, Namespace: ptr.To(TestNamespace)},
+		{Cr: &policiesv1beta1.PolicySetList{}, Namespace: ptr.To(TestNamespace)},
+		{Cr: &olmv1alpha1.CatalogSourceList{}, Namespace: ptr.To(TemporaryNamespace)},
 	}
```

Comment:
We will need a  unique list for hub and spoke.  And besides CR, is there a way to collect pod logs?
Also I assume any "List" actual collects individual CRs in yaml format? i.e., CGU list will allow us to check the specific cgu status, etc. 

For spoke, we need the following: 
- pod list
- namespace list
- precache/backup job(s)  (I'm not sure what is the NS, should probably check existing bug report or TALM repo or ask Jun)
- precache/backup pod log(s) (same here re NS)

Reply:
@yliu127 I've kept all the CRs in the same list here since it's not an error to try to list CRs that don't exist. But I can separate them into a list per hub/spoke if that's clearer. Same with the namespaces for pod logs.

But yes the list resource is just the way the reporter wants it and means we want to list all resources of that kind.

It uses the `ReporterNamespacesToDump` variable to determine which pod logs to dump. I'll have to update that to collect more relevant logs though since the namespaces we create won't have any pods.

Reply:
I think separate list is better because hub and spoke CRs are pretty much all different. 

Oh I see, in that case we should use cgu pod NS (openshift-operators) for pod log on hub. And for spoke, we need to find out which NS is used for precache/backup pods.

Reply:
Okay, I'll go ahead and separate them. The precache pod is already in the tsparams so I'll just use that namespace. I'll have to look for the backup pod though since it's not in the test